### PR TITLE
new example for using hibernate during import

### PR DESCRIPTION
**Pull request by mpheckel**
Hi Scott, I put a comment in the config class to explain what is happening.  Thanks

### CODE REVIEW

File: examples/hibernateImport/src/main/resources/hibernate.cfg.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE hibernate-configuration PUBLIC
+		"-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+		"http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+<hibernate-configuration>
+    <session-factory>
+      <property name="hibernate.connection.driver_class">org.hsqldb.jdbcDriver</property>  
+        <property name="hibernate.connection.url">jdbc:hsqldb:file:C:\Users\mheckel\workspace\ml-spring-batch\marklogic-spring-batch\examples\hibernateImport\data\hr</property>
```
Comment (sastafford) [inline examples/hibernateImport/src/main/resources/hibernate.cfg.xml]: the hibernate connection url is set to a directory on your computer and is causing the test to crash.  

if you set it to "jdbc:hsqldb:mem:test" then it seems to connect but then I get the error that I think you referred to previously:      

            java.lang.IllegalStateException: Cannot open an already opened ItemReader, call close first

File: examples/hibernateImport/data/hr.log [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/*C3*/SET SCHEMA PUBLIC
```
Comment (sastafford) [inline examples/hibernateImport/data/hr.log]: Go ahead and remove the entire data/ directory.  This is created when you open the HSQL Manager.  Don't need to check this in version control.

**Comment by mpheckel** [comment]
done

File: examples/hibernateImport/src/main/resources/hibernate.cfg.xml (hunk 1 above)
Comment (sastafford) [inline examples/hibernateImport/src/main/resources/hibernate.cfg.xml]: @mpheckel did you see this above comment?  If you run using an in-memory database, does your test case work or do you get an error?

**Comment by mpheckel** [comment]
I made the change but when I gradle importHumanResources there is an error thrown when it tries to instantiate the Step.



Besides changing the property for the URI, how do I run to make use of the in memory db?  Previously, I thought you said we were using the local db.. Do I use gradlew against the bat file?

**Pull request closed without merging**